Components/Queues/AsyncQueueConsumer.php

```diff
@@ -4,90 +4,146 @@
 
 namespace Smartbox\Integration\FrameworkBundle\Components\Queues;
 
-use Psr\Log\LoggerAwareInterface;
-use Psr\Log\LoggerAwareTrait;
-use Smartbox\Integration\FrameworkBundle\Components\Queues\Handler\AmqpQueueHandler;
-use Smartbox\Integration\FrameworkBundle\Core\Consumers\ConsumerInterface;
-use Smartbox\Integration\FrameworkBundle\Core\Consumers\IsStopableConsumer;
+use PhpAmqpLib\Message\AMQPMessage;
+use Smartbox\CoreBundle\Type\SerializableInterface;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\AsyncQueueDriverInterface;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\PhpAmqpLibDriver;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\QueueDriverInterface;
```

Comment:
QueueDriverInterface is not used in this case, so this reference can be deleted

Reply:
It's-a gone!